### DISCUSSION TITLE
Reclassification of Stimulants (cocaine, amph, etc) + Opioids

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#155] [Combination Chart]
Currently, on the web interface, amphetamines + opioids are classified as Caution with the message "Stimulants increase respiration rate allowing a higher dose of opiates. If the stimulant wears off first then the opiate may overcome the patient and cause respiratory arrest." However, cocaine + opioids returns the same message but categorizes the combination as Dangerous. The full chart also reflects these 2 classifications.

Via tripbot, stimulants + opioids does not return anything. Cocaine + opioids returns Dangerous as the classification with the same message as the web interface.

I think Caution would be the appropriate classification for these combinations with a message to keep doses within the person's normal tolerance range for each substance individually and an explanation of why/how the combo could become dangerous. Maybe something like:
Because these drugs diminish the effects of one another, some people may find themselves increasing their doses into a dangerous range. Particularly if the stimulant wears off before the opioid, it is possible that the person will overdose and experience respiratory depression. Be cautious with this combination and keep doses of each substance within your usual window of tolerance.

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/312
